Mapbox/MapboxMapsFoundation/Camera/CameraManager.swift

```diff
@@ -642,6 +644,30 @@ public class CameraManager {
         animationGroup.delegate = animationGroup
         cameraLayer.add(animationGroup, forKey: animationKey)
     }
+
+    /**
+        This function optimizes the bearing for set camera so that it is taking the shortest path
+
+        - Parameter startBearting: The current or start bearing of the map viewport
+        - Parameter endBearting: The bearing of where the map viewport should end at
+        - Returns: A `CLLocationDirection` that represents the correct final bearing accounting for positive and negatives
+    */
+    internal func optimizeBearing(startBearing: CLLocationDirection?, endBearing: CLLocationDirection?) -> CLLocationDirection? {
```

Comment:
nit: Change the formatting of these comments to match the other functions (Cmd+Option+/ is your friend :))